deny.toml

```diff
@@ -11,3 +11,6 @@ license-files = [{ path = "LICENSE", hash = 0xbd0eed23 }]
 [bans]
 multiple-versions = "warn"
 wildcards = "deny"
+
+[advisories]
+ignore = ["RUSTSEC-2024-0013"]
```

Comment:
Why not just upgrade libgit2?

Reply:
Sorry, missed your comment on the main thread. Closing.

Reply:
FYI we're dropping libgit2 soon (see https://github.com/helsing-ai/buffrs/pull/220)